lib/spack/spack/cmd/test_suite.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
+
+import os
+import platform
+import requests
+import spack.package
+import glob
+import argparse
+import llnl.util.tty as tty
+from spack.spec import Spec
+import spack.cmd.install as install    
+import spack.cmd.uninstall as uninstall
+import spack.util.spack_yaml as syaml
+import spack.compilers 
+import spack
+from yaml.error import MarkedYAMLError
+import jsonschema
+from jsonschema import Draft4Validator, validators
+from spack.error import SpackError
+import re
+import sys
+import time
+import traceback
+concreteTests = []
+
+
+description = "Compiles a list of tests from a yaml file. Runs Spec and concretize then produces cdash format."
+
+def setup_parser(subparser):
+    subparser.add_argument(
+        '-c', '--complete', action='store_true', dest='complete',
+        help='using this option switches from simple cdash output to compelet: simple is only build, complete is configure, build and test xml output.')
+    subparser.add_argument(
+        'yamlFile', nargs=argparse.REMAINDER,
+        help="yaml file that contains a list of tests, example yaml file can be found in /lib/spack/docs/tutorial/examples/test.yaml")
```

Comment:
Since you're adding a new command, can you add [tab completion](https://github.com/LLNL/spack/blob/develop/share/spack/spack-completion.bash) as well? It would look like:
```bash
function _spack_test-suite {
    if $list_options
    then
        compgen -W "-c --complete" -- "$cur"
    fi
}
```